src/components/trends/smbg/FocusedSMBGPointLabel.js

```diff
@@ -0,0 +1,134 @@
+    return null;
+  }
+
+  const {
+    bgUnits,
+    focusedPoint: { data, position, dayPoints, positions },
+    timePrefs,
+    lines,
+  } = props;
+
+  let parsedTime;
+  if (timePrefs.timezoneAware) {
+    if (data.time) {
+      parsedTime = Date.parse(data.time);
+    }
+  } else {
+    if (data.deviceTime) {
+      parsedTime = Date.parse(data.deviceTime);
+    }
+  }
+  const smbgTime = formatDisplayDate(parsedTime, timePrefs, 'dddd MMM D');
+  const shortDate = formatDisplayDate(parsedTime, timePrefs, 'MMM D');
+  const pointTooltips = _.map(dayPoints, (smbg, i) => (
+    <Tooltip
+      key={i}
+      content={<span className={styles.number}>{displayBgValue(smbg.value, bgUnits)}</span>}
+      position={positions[i]}
+      side={'bottom'}
+      tail={false}
+      offset={{ top: SIMPLE_VALUE_TOP_OFFSET, left: 0 }}
+    />
+  ));
+  let focusedTooltip;
+  if (lines) {
+    focusedTooltip = (<Tooltip
+      title={<span className={styles.explainerText}>{smbgTime}</span>}
+      position={position}
+      side={'right'}
+      offset={{ top: SIMPLE_DAY_TOP_OFFSET, left: SIMPLE_DAY_LEFT_OFFSET }}
+    />);
+  } else {
+    focusedTooltip = (<Tooltip
+      title={<span className={styles.tipWrapper}>
+        <span className={styles.shortDate}>{shortDate}</span>
+        <span className={styles.shortTime}>{millisecondsAsTimeOfDay(data.msPer24)}</span>
+      </span>
+      }
+      content={<span className={styles.tipWrapper}>
+        <span className={styles.detailNumber}>{displayBgValue(data.value, bgUnits)}</span>
+        <span className={styles.subType}>{data.subType}</span>
+      </span>
+      }
+      position={position}
+      side={'right'}
+      offset={{ top: 0, left: DETAILED_DAY_LEFT_OFFSET }}
+    />);
+  }
+  return (
+    <div className={styles.container}>
+      {pointTooltips}
+      {focusedTooltip}
+    </div>
+  );
+};
+
+FocusedSMBGPointLabel.defaultProps = {};
+
+FocusedSMBGPointLabel.propTypes = {
+  bgUnits: PropTypes.oneOf([MGDL_UNITS, MMOLL_UNITS]).isRequired,
+  focusedPoint: PropTypes.shape({
+    data: PropTypes.shape({
+      deviceTime: PropTypes.string,
+      msPer24: PropTypes.number.isRequired,
+      subType: PropTypes.string.isRequired,
+      time: PropTypes.string,
+      value: PropTypes.number.isRequired,
+    }).isRequired,
+    dayPoints: PropTypes.arrayOf(PropTypes.shape({
+      value: PropTypes.number.isRequired,
+    })).isRequired,
+    position: PropTypes.shape({
+      left: PropTypes.number.isRequired,
+      top: PropTypes.number.isRequired,
+    }).isRequired,
+    positions: PropTypes.arrayOf(PropTypes.shape({
+      left: PropTypes.number.isRequired,
+      top: PropTypes.number.isRequired,
+    })).isRequired,
+  }),
+  grouped: React.PropTypes.bool.isRequired,
+  lines: React.PropTypes.bool.isRequired,
+  timePrefs: PropTypes.shape({
+    timezoneAware: React.PropTypes.bool.isRequired,
+    timezoneName: React.PropTypes.oneOfType([React.PropTypes.string, null]),
```

Comment:
In the case that we are in timezone naive mode, the `timezoneName` is not provided and React will throw a warning. Actually, this is what's currently happening in the TrendsContainer in timezone naive mode because it has that property marked as `isRequired`. (https://github.com/tidepool-org/viz/blob/render-days/src/containers/trends/TrendsContainer.js#L65). So we might need to consider making it not `isRequired` everywhere or change the state handling so we always provide something for `timezoneName`, even in timezone-unaware mode. (Or maybe a `.oneOf([PropTypes.shape({}), PropTypes.shape({})])` which specifies both potential shapes, but that seems a little convoluted).